src/main/java/org/spongepowered/common/mixin/core/network/NetHandlerPlayServerMixin.java

```diff
@@ -811,12 +811,16 @@ public void processUseEntity(final CPacketUseEntity packetIn) {
         this.player.markPlayerActive();
 
         if (entity != null) {
-            final boolean flag = this.player.canEntityBeSeen(entity);
-            double d0 = 36.0D; // 6 blocks
-
-            if (!flag) {
```

Comment:
So I went ahead and looked at the [Forge patch for this class](https://github.com/MinecraftForge/MinecraftForge/blob/1.12.x/patches/minecraft/net/minecraft/network/NetHandlerPlayServer.java.patch) and this is _not_ one of the changes they make, `d0 = 36.0D` here is left alone. Why should we change the behaviour away from Forge here?

Reply:
Because there was this PR - https://github.com/MinecraftForge/MinecraftForge/pull/4736/ - that was headed at exactly this issue. Without the change the player sees that they can reach that entity, they swing the sword and.. nothing happens.
That PR wasn't even properly discussed - just staled and closed due to inactivity.